CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-23155: Kernel support packages and driver download functionality have been deprecated and will be removed in ACS 4.7 or later.
 - The field `error` returned for failed API calls has been deprecated, and it will be removed in a future release. Instead of using the `error` field, use the `message` field. The `message` field contains the same information as the `error` field.
 - The `/v1/summary/counts` API has been deprecated in 4.5 and will be removed in the future.
+- 'Dashboard' view under 'Vulnerability Management' is deprecated and will be removed in a future release. Use 'Workload CVEs', 'Exception Management', 'Platform CVEs', and 'Node CVEs' views instead.
 
 ### Technical Changes
```

Comment:
Can we add another line under this section, similar to:

```
- The ability to snooze Node and Platform CVEs is no longer enabled by default and can be enabled by setting `ROX_VULN_MGMT_LEGACY_SNOOZE` to `true` on Central.
```
with a reference to https://issues.redhat.com/browse/ROX-22251 if you think that is the most appropriate ticket.

Reply:
That makes sense. I have added it under `Technical Changes` instead of `Deprecated Features` section since the deprecation notice for node and platform CVE snooze was already sent in 4.3.